src/SingleMovieCard.js

```diff
@@ -8,8 +8,21 @@ const SingleMovieCard = ({ singleMovie, hideSingleView, displayMovieInfo}) => {
       <h1>{singleMovie.title}</h1>
```

Comment:
Because you have another `h1` on the page for the Rancid Tomatillos title, this should be a different element. There should be only one `h1` on the page at any given time.